step.yml

```diff
@@ -0,0 +1,117 @@
+title: Xcode Test without building
+summary: Tests compiled bundles.
+description: Tests compiled bundles by running `xcodebuild test-without-building` command.
+
+website: https://github.com/bitrise-steplib/bitrise-step-xcode-test-without-building
+source_code_url: https://github.com/bitrise-steplib/bitrise-step-xcode-test-without-building
+support_url: https://github.com/bitrise-steplib/bitrise-step-xcode-test-without-building/issues
+
+type_tags:
+- test
+
+is_always_run: false
+is_skippable: false
+run_if: ""
```

Comment:
Should this be included if it's empty?

Reply:
Not really, options with default values should not be included.

Reply:
Updated in: https://github.com/bitrise-steplib/bitrise-step-xcode-test-without-building/pull/1/commits/c3dc2da715c1971fbb7bb2ea39acacee6f3d9b12